README.md

```diff
@@ -38,17 +29,79 @@ I want to post a message (peep) to chitter
 As a maker
 So that I can see what others are saying  
 I want to see all peeps in reverse chronological order
+
+Still working on the following:
+-------------------------------
+As a maker
+So that I can better appreciate the context of a peep
+I want to see the time at which it was made
 ```
 
+Testing:
+--------
+
+Will be done with RSpec and Capybara
+
+Feature Tests:
+
+* User can sign up
+	
+	* As new user visiting site
+
+	* As long as they give a password and password confirmation that match
+
+	* But not with an email or username already registered
+
+* User can sign in
+
+	* With correct log in details
+
+	* But not with incorrect log in details
+
+* User can sign out
+
+	* After having been signed in
+
+* Peeps can be viewed on the homepage
+
+* User can post a peep
+
+	* Only if they're registered
+
+Unit Tests:
+
+* 
+
+General Structure:
+------------------
+
+Currently planning on having the following classes:
+
+* Peep - the posts that users create
+
+* User - stores the user's information
```

Comment:
This is all a good idea. Something I found worked well for students was having a series of MVPs (Minimum Viable Products) listed. That way, you can decide what constitute 'milestones'.

This could essentially be:

1. Posting and viewing peeps
2. Posting and viewing peeps as a logged in user
3. Filtering peeps by date (or something to extend it)